docs/changelogs/v2.10.0.md

```diff
@@ -0,0 +1,125 @@
+
+
+## Changelog
+
+> [!NOTE]
+> This is a Mainline Coder release. We advise enterprise customers without a staging environment to install our [latest stable release](https://github.com/coder/coder/releases/latest) while we refine this version. Learn more about how we release in our [docs](../install/releases.md).
+
+
+### BREAKING CHANGES
+
+- Removed `max_ttl` from templates (#12644) (@Emyrk)
+
+### Features
+
+- Make agent stats' cardinality configurable (#12535) (@dannykopping)
+- Upgrade tailscale fork to set TCP options for performance (#12574) (@spikecurtis)
+- Add AWS IAM RDS Database auth driver (#12566) (@f0ssel)
+- Support Windows containers in bootstrap script (#12662) (@kylecarbs)
+- Add `workspace_id` to `workspace_build` audit logs (#12718) (@sreya)
+- Make OAuth2 provider not enterprise-only (#12732) (@code-asher)
+- Allow number options with monotonic validation (#12726) (@dannykopping)
+- Expose workspace statuses (with details) as a prometheus metric (#12762) (@dannykopping)
+- Support adjusting child proc OOM scores (#12655) (@sreya)
```

Comment:
Can you explain what the benefit is to the user and why they may want to turn this on, or if it is purely automatic

Reply:
(the OOM score one)

Reply:
I honestly don't know what this does.